<!--
  Copyright 2022 VMware, Inc.
  SPDX-License-Identifier: BSD-2
-->
<p>Inclusive Terminology Scanner</p>
<h2>Noncompliant Code Example</h2>
<pre>
Master example of blacklist 
</pre>
<h2>Compliant Solution</h2>
<pre>
Primary example of denylist
</pre>
